__tests__/actionCreators/resources.test.js

```diff
@@ -15,12 +16,19 @@ import _ from 'lodash'
 
 jest.mock('sinopiaServer')
 
+const orgConsoleError = console.error
+
 beforeEach(() => {
   sinopiaServer.foundResourceTemplate.mockResolvedValue(true)
   sinopiaServer.getResourceTemplate.mockImplementation(getFixtureResourceTemplate)
   sinopiaServer.loadRDFResource.mockImplementation(getFixtureResource)
   let counter = 0
   shortid.generate = jest.fn().mockImplementation(() => `abc${counter++}`)
+  console.error = jest.fn()
+})
+
+afterEach(() => {
+  console.error = orgConsoleError
```

Comment:
Could this be renamed to something like 'resetConsoleErrorCache' or add a short comment about why this is done?

Reply:
Sure, I like `resetConsoleErrorCache` as a better descriptor. I'll also add a short comment to explain that is to avoid unnecessary console errors when we're testing raising errors.